Immutable.Net/Immutable.Net/Immutable`T.cs

```diff
@@ -59,7 +64,9 @@ private static class Accessor<TValue>
         /// </summary>
         public Immutable() 
         {
-            self = new T();
+            createDelegate = Expression.Lambda<Func<T>>(Expression.New(typeof(T))).Compile();
```

Comment:
If the createDelegate already exists, we don't need to recreate it here. We should reuse this as well.